src/main/java/model/datafeed/DatafeedV2.java

```diff
@@ -0,0 +1,26 @@
+package model.datafeed;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties()
+public class DatafeedV2 {
```

Comment:
We could start using lombok for getter/setter (it seems it started to get used in AgentInfo for instance)